compiler/qsc_ast/src/ast.rs

```diff
@@ -1513,6 +1546,12 @@ impl From<Idents> for Vec<Ident> {
     }
 }
 
+impl From<&Idents> for Vec<Ident> {
```

Comment:
Yes I think this is worth it to have, as the body of both preserves the origin and makes copies, having one that is for non-ref and one that is for ref makes for ease of use of this conversion elsewhere in the code. It is difficult to have to try and deref a reference when calling these because then rust complains about moving values.

Reply:
Does calling `clone` and then `into` cause the underlying idents to be cloned twice?

Reply:
Ah okay, I've changed that to an `into_vec`. Okay I've removed the `From<&Idents> for Vec<Ident>` and am using `.clone().into()` at the call sites.